### ISSUE TITLE
I would like to recommend adding the HTML5 work done by xemlock

### ISSUE THREAD

**Issue by ayumihamsaki** [mewebstudio/Purifier#108]
Would it be a good idea to add both the packages:

- ezyang/htmlpurifier

- xemlock/htmlpurifier-html5

To this lib. The work done by xemlock is really good and he updates his repo all the time!

As per conversation: https://github.com/xemlock/htmlpurifier-html5/issues/49

**Comment by AugmentBLU** [comment]
Agreed, the other package is falling behind and isn't working with many HTML5 elements. It's causing all sorts of issues for me now.

**Comment by GJPhilippi** [comment]
@imannms No,  it doesn't seem so. But, with the published configuration a set of custom tags and attributes are added into the Purifier config, which are HTML5 elements.
